update_pr.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import requests
+
+def process_github_event(comment_url, issue_url, github_token, pr_title, pr_body, jira_key):
+    """
+    Add a rocket reaction to a comment and update a pull request title and body with a JIRA key.
+    """
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "Authorization": f"Bearer {github_token}"
+    }
+
+    # Add a rocket reaction to the comment
+    reaction_url = f"{comment_url}/reactions"
+    reaction_payload = {"content": "rocket"}
+    reaction_response = requests.post(
+        reaction_url,
+        headers={**headers, "Accept": "application/vnd.github.squirrel-girl-preview+json"},
```

Comment:
This weird Accept header was needed only when this API was in preview, see [this post](https://developer.github.com/changes/2016-05-12-reactions-api-preview/). According to the [up-to-date API documentation](https://docs.github.com/en/rest/reactions/reactions?apiVersion=2022-11-28#create-reaction-for-an-issue-comment), this is no longer needed, and much saner `Accept: application/vnd.github+json` is enough. Might be worth adding the API version header as well: `X-GitHub-Api-Version: 2022-11-28`.